src/main/scala/tech/cryptonomic/conseil/routes/openapi/OpenApiDoc.scala

```diff
@@ -52,14 +52,21 @@ object OpenApiDoc extends DataEndpoints
 
   /** Function for validation definition in documentation which appends DocumentedResponse to the list of possible results from the query.
     * In this case if query fails to validate it will return 400 Bad Request.
-    *  */
-  override def validated[A](response: List[OpenApiDoc.DocumentedResponse], invalidDocs: Documentation): List[OpenApiDoc.DocumentedResponse] =
+    * */
+  override def validated(response: List[OpenApiDoc.DocumentedResponse], invalidDocs: Documentation): List[OpenApiDoc.DocumentedResponse] =
     response :+ OpenApiDoc.DocumentedResponse(
       status = 400,
       documentation = invalidDocs.getOrElse(""),
       content = Map(
         "application/json" -> MediaType(schema = Some(Schema.Array(Schema.simpleString)))
       )
+    ) :+ OpenApiDoc.DocumentedResponse(
```

Comment:
I'm a bit lost here, is there an easy way to understand what's happening and when each response will be given?

Reply:
basically its definition for the OpenApi(and swagger) -> it describes what is expected by each response. It is easier to understand how it works using swagger, when you can chose `accept` header for the request and it shows curl request used.